content/news/march-2016-tournament-announcement.md

```diff
@@ -0,0 +1,13 @@
+It is time for yet another OpenRA tournament, which, for the first time ever, will feature a prize money of $200!
```

Comment:
Also, will feature a monetary prize/reward of, or just say it will feature a prize.